src/plugins/Dashboard/index.js

```diff
@@ -201,6 +201,20 @@ module.exports = class Dashboard extends Plugin {
     }
   }
 
+  scrollBehaviour (toggle) {
+    if (!this.opts.disablePageScrollWhenModalOpen) return
+    const body = document.querySelector('body')
+    switch (toggle) {
+      case 'enable':
+        Object.assign(body.style, { overflow: 'initial', height: 'initial' })
```

Comment:
not sure about the implications of doing this…eg if i have set `overflow: hidden` on the body for some reason in my own css, this might change it once the modal closes. I feel like adding and removing a CSS class with `!important` rules from the body might be more reliable?

Reply:
well, I suppose you could just set the option to false if you are already doing `overflow: hidden` anyway.

Reply:
You are right, there’s no advantage in using style object. Thinking if `!important` is an overkill.